boot/makebootable_test.go

```diff
@@ -564,6 +564,56 @@ current_trusted_recovery_boot_assets={"bootx64.efi":["39efae6545f16e39633fbfbef0
 
 	// make sure SealKey was called
 	c.Check(sealKeyCalls, Equals, 1)
+
+	pbc, err := boot.BootChainsFromFile(filepath.Join(dirs.SnapFDEDirUnder(boot.InstallHostWritableDir), "boot-chains"))
+	c.Assert(err, IsNil)
+	c.Check(pbc, DeepEquals, boot.PredictableBootChains{
+		boot.BootChain{
+			BrandID:        "my-brand",
+			Model:          "my-model-uc20",
+			Grade:          "dangerous",
+			ModelSignKeyID: "Jv8_JiHiIzJVcO9M55pPdqSDWUvuhfDIBJUS-3VW7F_idjix7Ffn5qMxB21ZQuij",
+			AssetChain: []boot.BootAsset{
+				{
+					Role:   "recovery",
+					Name:   "bootx64.efi",
+					Hashes: []string{"39efae6545f16e39633fbfbef0d5e9fdd45a25d7df8764978ce4d81f255b038046a38d9855e42e5c7c4024e153fd2e37"},
+				}, {
+					Role:   "recovery",
+					Name:   "grubx64.efi",
+					Hashes: []string{"aa3c1a83e74bf6dd40dd64e5c5bd1971d75cdf55515b23b9eb379f66bf43d4661d22c4b8cf7d7a982d2013ab65c1c4c5"},
+				},
+			},
+			Kernel:         "pc-kernel",
+			KernelRevision: "",
```

Comment:
this feels wrong and unexpected, I think there's a typo above in the setup where we should have `snap.Revision{N:1}` and we actually have `snap.Revision{N:0}` at line 449

Reply:
Changed to -1 (unasserted)

Reply:
Finally changed to "1" to match the file name as @pedronis suggested.